src/espeak.c

```diff
@@ -203,6 +204,10 @@ static void set_alsa_volume(int vol)
 	 * 100%. */
 
 	int volume = (vol + 1) * 50 / 10 + 50;
+	/*lock the volume to the user-set volume if specified*/
+	if(volumeSet){
+		volume = defaultVolume;
```

Comment:
Mmm, but then the volume is locked to that value, and one cannot use the speakup keybord shortcut to change it (capslock+1 or +2) any more.

Actually, I'm surprised that you say you couldn't find a way to adjust the volume. Did you just not know the speakup shortcut? If we add a volume parameter, then people will quickly request a rate parameter, a pitch parameter, etc. while this should all be in control by speakup, like done for the hardware synthesizers. If we want to make the default volume configurable, that should rather be done through scripts to set default speakup parameters, and then all speakup parameters will be configurable that way